### PR TITLE
Updated d3-geo to reflect version 1.9.0: added fitWidth and fitHeight…

### LINKED ISSUE #20325: [d3-geo] Update to Minor Version 1.9
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @tomwanzek @Ledragon 

@Ledragon Just when you thought you were off the hook... You're it! 😉 

The changes with this minor release are small (`projections.fitWidth/fitHeight`). Whenever you  can get to it. I will do the review.

### PR DESCRIPTION
**Pull request by Ledragon**
Added **projection.fitWidth** and **projection.fitHeight** methods.

Closes #20325 
@tomwanzek The floor is yours :-)

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [v1.9.0](https://github.com/d3/d3-geo/releases/tag/v1.9.0)
- [x] Increase the version number in the header if appropriate.

### CODE REVIEW

**Comment by Ledragon** [comment]
Should be fine; I cannot manage to get the linter to work properly on my machine, the `strictFunctionTypes` option does not seem to be recognized. Any hint on how to fix this?
Using npm v5.3.0 and node v8.4.0 on Windows 10

File: types/d3-geo/index.d.ts [hunk 1]
```diff
@@ -804,7 +804,62 @@ export interface GeoProjection extends GeoStreamWrapper {
      * @param object A geographic geometry collection supported by d3-geo (An extension of GeoJSON geometry collection).
      */
     fitSize(size: [number, number], object: ExtendedGeometryCollection<GeoGeometryObjects>): this;
-
+    /**
```
Comment (tomwanzek) [inline types/d3-geo/index.d.ts]: Can you add a blank line between the `fitSize`, `fitWidth`, and `fitHeight` related definitions? Not critical, but the sectioned off readability going forward is a bit better. Seeing that d3-geo has very lengthy definitions.

**Review by tomwanzek — CHANGES_REQUESTED** [review]
Other than the little formatting suggestion I remarked on below, the definitions are o.k. The only thing would be to update the tests to include `fitWdith(...)` and `fitHeight(...)`.

Otherwise: thanks already!!!

**Comment by Ledragon** [comment]
IDK what was wrong with me yesterday, I got it all wrong in any way I can... It should be fine by now

**Review by tomwanzek — APPROVED** [review]
@LeDragon Thanks for the PR. LGTM

**Pull request merged**